protostar/cheatable_starknet/cheatable_entry_point.py

```diff
@@ -280,3 +300,11 @@ def execute(
             general_config=general_config,
             tx_execution_context=tx_execution_context,
         )
+
+    def _map_stark_exception(self, stark_exception: StarkException):
+        results = re.findall("Error message: (.*)", stark_exception.message or "")
+        if len(results) > 0:
+            return CheatableException(message=results[0], raw_ex=stark_exception)
```

Comment:
Why are we poppin' just one element here?

Reply:
We need to operate on error codes, so it must be a one message. This code will change anyway once Starknet will be integrated with Cairo 1.